java/flight/flight-core/src/test/java/org/apache/arrow/flight/TestClientMiddleware.java

```diff
@@ -246,9 +246,11 @@ public FlightClientMiddleware onCallStarted(CallInfo info) {
   }
 
   // Used to test that middleware can send and receive multi-valued text and binary headers.
+  @SuppressWarnings({"DoubleBraceInitialization"})
```

Comment:
Maybe just initialize these values (here and below) the old-fashioned way, rather than add the SuppressWarnings?

Reply:
Thanks, changed